src/NServiceBus.Core.Tests/DeliveryConstraintContextExtensions/DeliveryConstraintContextExtensionsTests.cs

```diff
@@ -22,7 +22,7 @@ public void Should_be_able_to_determine_if_delivery_constraint_is_supported()
             settings.Set<TransportDefinition>(fakeTransportDefinition);
             settings.Set<TransportInfrastructure>(fakeTransportDefinition.Initialize(settings, null));
 
-            var context = new FeatureConfigurationContext(settings, null, null);
+            var context = new FeatureConfigurationContext(settings, null, null, null);
```

Comment:
This is going to become a very nasty ctor as we proceed but I guess we'll see what happens when more components are added